package-lock.json

```diff
@@ -1,8 +1,13671 @@
 {
```

Comment:
@brian-frederick looks like rally links are determined to reside in here - I see the edits made at the bottom but it looks like the old relations were injected in? Not sure if this is how you had intended it

Reply:
yea so it looks like my default registry on my mac is set up to be rally's. It's a pain but I discovered I can specify the registry like so when i run installations
`npm install --registry=https://registry.npmjs.org/`

Reply:
So I can clean this up in a future pr @whob13